### DISCUSSION TITLE
virtnbdbackup | bash script | cron mail [32 and [0m

### DISCUSSION THREAD

**Discussion by slu-010101** [abbbi/virtnbdbackup#200] [General]
Hi,

I start virtnbdbackup with my bash script who send me an mail, but every word start with "[32" and end with "[0m".
Never see that before, any idea why this happen?

```
grep -v "#" /root/vmlist.txt | while read VMGUEST ; do
   virtnbdbackup --noprogress --scratchdir "${TMPVIRTNBDBACKUP}" -d "${VMGUEST}" -l auto -o "${FULLVMBACKUPPATH}"
done
```

Example:
```
[32m[2024-08-06 02:50:02][0m[34m INFO[0m root virtnbdbackup - main [main]:  Backup level: [auto][0m
[32m[2024-08-06 02:50:02][0m[34m INFO[0m root virtnbdbackup - main [main]:  Libvirt library version: [9000000][0m
[32m[2024-08-06 02:50:02][0m[34m INFO[0m root virtnbdbackup - main [main]:  NBD library version: [1.14.2][0m
```

Debian 12.6
virtnbdbackup 2.9-1

**Comment by abbbi** [comment]
these are ansi escape codes for the colors from the log library, use **--nocolor** option for non colored output or export NOCOLOR environment variable to disable colored log output. Seems like you are using a terminal which does not support these escape codes Or your Mail viewer doesnt Interpret them

**Comment by slu-010101** [comment]
This make sense, the mail has this content type:
```
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
```
I add --nocolor as parameter in my script, thanks for the hint.
